src/transforms/ppmt.jl

```diff
@@ -0,0 +1,171 @@
+using EmpiricalCDFs
+using Distributions
+using Optim
+
+"""
+    Ppmt()
+
+Applies the projection pursuit multivariate transform to the table. 
+Taken out from the documentation: Friedman, J. H. (1987). 
+Exploratory Projection Pursuit. Journal of the American Statistical 
+Association, 82(397), 249–266. https://doi.org/10.2307/2289161
+
+"""
+
+struct Ppmt
+    table
+    tol::Real
+    perc::Real
+    trunc::Integer
+    sample::Integer
```

Comment:
We use two whitespaces as indentation, please follow the code style.

Reply:
Also, notice that the table itself is not part of the transform parameters. You need to reduce the set of parameters to a minimum set that users can expect to change in final scripts.